app/src/main/java/com/example/caloriecounter/ui/BottomBar.kt

```diff
@@ -29,8 +26,8 @@ fun BottomBar(navController: NavHostController) {
 
     BottomNavigation(
         modifier = Modifier.navigationBarsPadding(),
-        backgroundColor = Color.White,
-        contentColor = Color.Blue
+        backgroundColor = MaterialTheme.colors.background,
+        contentColor = MaterialTheme.colors.primary
```

Comment:
nit: if you will decide to use M3 I bet `BottomNavigation` should use next colors
https://m3.material.io/components/navigation-bar/specs